packages/core/admin/admin/src/content-manager/components/Form.tsx

```diff
@@ -119,47 +135,61 @@ const Form = React.forwardRef<HTMLFormElement, FormProps>(
       }
     }, [props.initialValues]);
 
+    const setErrors = React.useCallback((errors: FormErrors) => {
+      dispatch({
+        type: 'SET_ERRORS',
+        payload: errors,
+      });
+    }, []);
+
     /**
      * Uses the provided validation schema
      */
-    const validate: FormContextValue['validate'] = React.useCallback(async () => {
-      if (!props.validationSchema) {
-        return null;
-      }
+    const validate: FormContextValue['validate'] = React.useCallback(
+      async (shouldSetErrors: boolean = true) => {
```

Comment:
in which case you would call the validate without setting the errors?

Reply:
Perhaps in the case of bulk actions where they still validate 🤔